clang/include/clang/Basic/DiagnosticCommonKinds.td

```diff
@@ -234,6 +234,9 @@ def err_cxx23_size_t_suffix: Error<
 def err_size_t_literal_too_large: Error<
   "%select{signed |}0'size_t' literal is out of range of possible "
   "%select{signed |}0'size_t' values">;
+def ext_cpp_bitint_suffix : Extension<
+  "'_BitInt' suffix for literals is a Clang extension">,
```

Comment:
Just noticed that the corresponding C warnings don’t display the suffix either, so if that ends up being too complicated, then that’s not that too important either; it would just be nice to have imo.